packages/shared/lib/services/connection.service.ts

```diff
@@ -821,11 +821,13 @@ class ConnectionService {
                     expiresAt = parseTokenExpirationDate(rawCreds['expires_at']);
                 } else if (rawCreds['expires_in']) {
                     expiresAt = new Date(Date.now() + Number.parseInt(rawCreds['expires_in'], 10) * 1000);
+                } else {
+                    expiresAt = new Date(Date.now() + 55 * 60 * 1000);
```

Comment:
@bodinsamuel, just updated the PR. I have defined it within the function itself so that `DEFAULT_EXPIRES_AT` can reflect the current time dynamically each time it's accessed.

Reply:
what I mean is that you can put this expiration as a constant without changing the code
`expiresAt = new Date(Date.now() + DEFAULT_EXPIRES_AT_MS);`

Reply:
ah yes 💯, updated in [here](https://github.com/NangoHQ/nango/pull/2469/commits/0c7c385da0e99787a7a880e7bc9957bf829ef937)